### DISCUSSION TITLE
Multiple dashboards in Pulse

### DISCUSSION THREAD

**Discussion by jspekken** [laravel/framework#49411] [General]
Hi there,

Love the new Laravel Pulse package but I was wondering if there are any plans to allow multiple dashboards. We'd like to run pulse on several screens in our office but right now we have to scroll to a specific section on the dashboard itself so that every screen displays separate information.

**Comment by chuckrincon** [comment]
Pulse is a package created with Livewire, so you can extend it as much as you want.

For example, to only show queues:

Create a new dashboard component `app/Livewire/Dashboards/Queues.php`
```bash
artisan make:livewire Dashboards/Queues 
```

Register the component route `routes/web.php`
```php
Route::get('/pulse/queues', Dashboards\Queues::class);
```

Include the cards in the view `views/livewire/dashboards/queues.blade.php`
```blade
<x-pulse>
    <livewire:pulse.queues cols="4" />
</x-pulse>

```
